### ISSUE TITLE
Bot should warn developers once when branches do not match convention

### ISSUE THREAD

**Issue by jack828** [microadam/exso#22]
If a developer creates a branch that does not start with one of the known conventions e.g. feat/ feature/ fix/ bug/ then when these get merged into releases it creates additional overhead and confusion for PMs releasing the work, as their release commands are ignored.

It would be good if when PRs are created and the various status checks are added, that it could do a quick check of the branch naming and warn the developer (once) that it does not match the naming convention for releases.

e.g. 

```
@jack828 Your branch name starts with `foobar/` which is not one of the supported branch names for the release workflow.
If deployments of this branch are required, please recreate the pull request with one of the following prefixes:
Features - `feature/` or `feat/`
Bugfixes - `fix/` or `bug/`
```

Related https://github.com/microadam/exso/issues/12

My thoughts above moving this from the release PR comment to the work PR comment is that it is the developer's responsibility to make the branch compatible with releases.

**Comment by microadam** [comment]
Yeah its a good idea, as you can see from #12 its an issue I have known about for some time! Doing it just ONCE might be tricky though, as the app is completely stateless
